message_ix_buildings/chilled/preprocess/archetypes.py

```diff
@@ -0,0 +1,157 @@
+"""
+Create buildings archetypes maps using specified inputs
+"""
+
+import datetime
+import os
+from itertools import product
+
+import numpy as np
+import xarray as xr
+from functions.variable_dicts import VARS_ARCHETYPES  # type: ignore
+from preprocess.message_raster import create_message_raster  # type: ignore
+
+from message_ix_buildings.chilled.util.config import Config  # type: ignore
+from message_ix_buildings.chilled.util.util import (
+    get_archs,
+    read_arch_inputs_df,
+    read_arch_reg_df,
+)
+
+
+def create_archetypes(config: "Config"):
+    out_path = os.path.join(config.project_path, "out", "version")
+    archetype_path = os.path.join(out_path, config.vstr, "rasters")
+
+    # if archetypes folder does not exist, create it
+    if not os.path.exists(archetype_path):
+        os.makedirs(archetype_path)
+
+    # get raster file and message map
+    country_ras, reg_ras, map_reg, iso_attrs = create_message_raster(config)
+
+    # get archs
+    vers_archs = get_archs(config)
+
+    # save MESSAGE regions map
+    msg_file = "map_reg_MESSAGE_" + config.node + ".nc"
+    map_reg.to_netcdf(os.path.join(archetype_path, msg_file))
+    print(
```

Comment:
Just a note for the future: such information is best printed via [python's logging utility](https://docs.python.org/3/howto/logging.html#logging-basic-tutorial). Please consider improving that in the future (and you could track this with an issue, too, if you'd like).

Reply:
Thanks! That's a very good point -- I have created issue #24 to track this.